srcd/superset/assets/stylesheets/less/custom-brand.less

```diff
@@ -566,16 +568,32 @@ h4.panel-title {
   font-size: 14px !important;
   border-radius: 0.2rem;
   border: none;
+  width: 124px;
```

Comment:
If I understand this right, this is forcing the "Run query" and "Stop" to have the same width, right?
I think it's a bit risky to set a fixed width. For example for me "Run query" with 124px has a little bit of less margin on the right side.
But what is important, this width is based on the words "Run query", which can change if you set a different language. Try for example russian or brazilian portuguese, the text overflows the button:
![image](https://user-images.githubusercontent.com/1469173/60713490-21181e80-9f11-11e9-87df-739db12bfa99.png).

I don't know what's the better approach for this. Maybe a `min-width` will be a good compromise, it makes "Run query" and "Stop" have the same width, while allowing the button to grow if it contains longer text translations.

I think this fixed `width` is also present for other buttons in this PR, so this will probably apply in more places.

Reply:
Thanks @carlos! I already pushed a commit `40f95cc` with `min-width`.